### PR TITLE
Vapor 2 support

### LINKED ISSUE #25: 2.0.0 support?
I'm trying to test out 2.0alpha; I'm currently blocked with no 2.0.0 alpha release for postgresql-provider. Any idea when it will be added?

Thanks.

### PR DESCRIPTION
**Pull request by vzsg**
This is the continuation of #26.
Resolves #25.

### CODE REVIEW

**Comment by ChasonTang** [comment]
It is seem that the tests are broken.The module name of `@testable import PostgreSQLDriverTests` is wrong

**Comment by vzsg** [comment]
Where do you see that line? All tests have `import FluentPostgreSQL` where necessary in this branch. Anyhow, the Fluent test suite currently fails due to missing parts.

One of the current problems is the missing support for timestamps from the wire protocol.  
It's a work in progress.

File: Tests/LinuxMain.swift [hunk 1]
```diff
@@ -1,13 +1,10 @@
 #if os(Linux)
 
 import XCTest
-@testable import FluentPostgreSQLTests
+@testable import PostgreSQLDriverTests
```
Comment (ChasonTang) [inline Tests/LinuxMain.swift]: PostgreSQLDriverTests is incorrect module name

Comment (ChasonTang) [reply Tests/LinuxMain.swift]: @vzsg

Comment (vzsg) [reply Tests/LinuxMain.swift]: Great catch, thanks.

Comment (ChasonTang) [reply Tests/LinuxMain.swift]: Perhaps we can open an issue to present the progress that porting PostgreSQLDriver to vapor2

**Pull request merged**